Discovering/Property/UseCases/Property/AddPropertyForAdvisor/AddPropertyForAdvisorRequest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\RealEstate\Discovering\Property\UseCases\Property\AddPropertyForAdvisor;
+
+class AddPropertyForAdvisorRequest
```

Comment:
Consider handling the type casting of input values in this class instead of doing it in the Controller/Command class.
Also, properties could be public